build.go

```diff
@@ -83,32 +81,36 @@ func Build(
 
 		targetsLayer.Metadata = map[string]interface{}{
 			"built_at": clock.Now().Format(time.RFC3339Nano),
-			"command":  command,
-		}
-
-		command, ok := targetsLayer.Metadata["command"].(string)
```

Comment:
@ForestEckhardt do you know why this code existed before or when you could ever get into the failure case where `command` would not be defined as a string?

Reply:
We use to reuse the binary layer from cache if nothing changed  workspace directory. We no longer do that and always rebuild from source. This was metadata that we used to rewrite the start command on a rebuild where we reused the binary layer. That functionality has been removed so this is just dead code.